src/Uno.UI.Wasm/ts/Windows/Devices/Midi/MidiOutPort.ts

```diff
@@ -0,0 +1,10 @@
+﻿namespace Windows.Devices.Midi {
+	export class MidiOutPort {
+		public static sendBuffer(encodedDeviceId: string, timestamp: number, ...args: number[]) {
+			var midi = Uno.Devices.Midi.Internal.WasmMidiAccess.getMidi();
+			var deviceId = decodeURIComponent(encodedDeviceId);
+			var output = midi.outputs.get(deviceId);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'output' is never reassigned; use 'const' instead of 'var'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5305560)